shell/utils/__tests__/formatter.test.ts

```diff
@@ -0,0 +1,143 @@
+import { formatEncryptionSecretNames } from '@shell/utils/formatter';
+
+describe('formatter', () => {
+  const secrets = [
```

Comment:
Could you please filter up the used object to just the relevant properties used for this computation?
A whole object makes very complicate to understand the aim of the test and the involved data.